modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js

```diff
@@ -272,18 +272,10 @@ export default class Blogs {
 
 	_getValuesByName(name) {
 		const nodes = document.querySelectorAll(
-			`input[name*=${this._config.namespace + name}]`
+			`input[name=${this._config.namespace + name}]`
```

Comment:
It would be even better with "^":
`input[name^=${this._config.namespace}${name}]`